storage/innobase/include/fil0crypt.h

```diff
@@ -612,6 +612,58 @@ fil_space_get_scrub_status(
 	const fil_space_t*		space,
 	fil_space_scrub_status_t*	status);
 
+struct redo_log_key {
+	uint  version;
+	char  key[ENCRYPTION_KEY_LEN];
+	ulint read_count;
+	ulint write_count;
+	bool  present;
+
+	bool
+	persisted() const {
+		return version != 0;
+	}
+};
+
+/** Handles the fetching/generation/storing/etc of keyring redo log keys.
+
+ This class is *NOT* thread safe, as thread safety is not required.
+ Data is only accessed/modified on the following points:
+ * When the redo space is created, at startup
```

Comment:
Remove the "*" at beginning of this line and below

Reply:
That's a markdown style list there.